src/main/java/ui/pages/DemoDashboardPage.java

```diff
@@ -0,0 +1,62 @@
+package ui.pages;
+
+import constants.Constants;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import ui.driver.DriverManager;
+
+import java.util.List;
+
+import static ui.helpers.ActionsHelper.resizeSquare;
+import static ui.pages.HomePage.HOME_URL;
+
+public class DemoDashboardPage extends BasePage {
```

Comment:
In the PO structure, we use at least 3 layers: page, helper, and test layers. You can clarify if it is necessary for you now or just page-tests are enough.

Reply:
So, as we discussed yesterday, suppose I can leave it like it is now.